content/english/pygame-pong/activity-2.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Activity 2 - Change the color of objects in game."
+date: 2019-07-29T13:24:17-07:00
+weight: 4
+draft: false
+---
+
+Great! You juts fixed the game's boundary. Now, notice that how everything inside game looks black and white. Let's give it some colors. 
```

Comment:
In the code, can we use COLOR as the attribute name, and consistently use 'color' (American English spelling) everywhere?

Reply:
Yes, will change